TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -220,10 +220,12 @@ private void QuantificationAnalysis()
             {
                 spectraFileInfo = new List<SpectraFileInfo>();
 
-                foreach (var file in Parameters.CurrentRawFileList)
+                for (int i = 0; i < Parameters.CurrentRawFileList.Count; i++)
                 {
+                    var file = Parameters.CurrentRawFileList[i];
+
                     // experimental design info passed in here for each spectra file
-                    spectraFileInfo.Add(new SpectraFileInfo(fullFilePathWithExtension: file, condition: "", biorep: 0, fraction: 0, techrep: 0));
+                    spectraFileInfo.Add(new SpectraFileInfo(fullFilePathWithExtension: file, condition: "", biorep: i, fraction: 0, techrep: 0));
```

Comment:
can replace "file" with "Parameters.CurrentRawFileList[i]" so that you don't have to create an instance of "file" in memory.
We only use the variable once, so faster to look it up on the fly.

Reply:
I prefer the clarity of the way I wrote it and I don't think the time or memory usage for this code block is meaningful